salt/reactors/mitxpro/edxapp_highstate.sls

```diff
@@ -0,0 +1,4 @@
+edxapp_highstate:
+  local.state.apply:
+    - tgt: 'edx-*mitxpro-production-xpro-production-*'
+    - queue: True
```

Comment:
I think it's worth either modifying pillar data that applies only to the non-base instances, or appending pillar data here to specify the ansible flags to only run the configuration updates.